src/Mobs/PassiveMonster.cpp

```diff
@@ -133,7 +138,7 @@ void cPassiveMonster::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 				virtual bool Item(cEntity * a_Entity) override
 				{
-					// if we're the same species as someone around and they dont have a partner, swipe right
+					// if we're the same species as someone around and they don't have a partner, swipe right.
```

Comment:
I have a feeling @tigerw will come up with a joke and post it here. Anyways, I'll change it for something less urban.